test/api/test-ot-face.c

```diff
@@ -109,6 +131,14 @@ test_face (hb_face_t *face,
   hb_ot_var_normalize_variations (face, NULL, 0, NULL, 0);
   hb_ot_var_normalize_coords (face, 0, NULL, NULL);
 
+  hb_ot_glyph_decompose_funcs_t funcs;
+  funcs.move_to = (hb_ot_glyph_decompose_move_to_func_t) move_to;
+  funcs.line_to = (hb_ot_glyph_decompose_line_to_func_t) line_to;
+  funcs.conic_to = (hb_ot_glyph_decompose_conic_to_func_t) conic_to;
+  funcs.cubic_to = (hb_ot_glyph_decompose_cubic_to_func_t) cubic_to;
+  for (unsigned gid = 0; gid < 10; ++gid)
```

Comment:
I like the new API (I haven’t tested it yet), but for consistency with the rest of HarfBuzz API, this should be modeled after the font functions API (having create and set methods and making the struct opaque).

Reply:
Oh, thank you so much, will apply.

Reply:
Done, thanks a lot for the suggestion, please have a look.